treeherder/webapp/api/performance_data.py

```diff
@@ -898,6 +956,45 @@ def _get_signatures(self, repository_name, framework, interval, no_subtests):
         signatures = self._get_signatures_values(signatures)
         return signatures
 
+    def _create_graph_links(
+        self,
+        base_repo_name,
+        new_repo_name,
+        base_revision,
+        new_revision,
+        framework,
+        time_range,
+        signature,
+    ):
+        highlighted_revision_key = 'highlightedRevisions'
+        time_range_key = 'timerange'
+        series_key = 'series'
+
+        highlighted_revisions_params = [
+            (highlighted_revision_key, new_revision[:12]),
+            (highlighted_revision_key, base_revision[:12]),
+        ]
+
+        graph_link = 'graphs?%s' % urlencode(highlighted_revisions_params)
```

Comment:
```suggestion
        graph_link = 'https://treeherder.mozilla.org/perfherder/graphs?%s' % urlencode(highlighted_revisions_params)
```

Reply:
hey Beatrice, I thought we agreed to do without the beginner part of the url, what changed?